worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeBiomeRegistry.java

```diff
@@ -72,12 +72,13 @@ static void populate() {
         Map<Integer, BiomeGenBase> biomes = HashBiMap.create();
         Map<Integer, BiomeData> biomeData = new HashMap<Integer, BiomeData>();
 
-        for (BiomeGenBase biome : BiomeGenBase.getBiomeGenArray()) {
+        for (BiomeGenBase biome : BiomeGenBase.biomeRegistry) {
             if ((biome == null) || (biomes.containsValue(biome))) {
                 continue;
             }
-            biomes.put(biome.biomeID, biome);
-            biomeData.put(biome.biomeID, new ForgeBiomeData(biome));
+            int biomeId = BiomeGenBase.getIdForBiome(biome);
```

Comment:
Is the `populate` method really still needed? Most of this can be done now directly using the `getIdForBiome` or `getObjectFromId` methods. E.g. I did it like this: https://github.com/Minecrell/worldedit-spongevanilla/commit/2bf1607abf97538daeefdaa48a040bc369b73f7e#diff-b9960f83cb3bc0ea696930900a3e25b9L40